src/Azure/AzureAD/test/FunctionalTests/WebAuthenticationTests.cs

```diff
@@ -31,14 +31,8 @@ public WebAuthenticationTests(WebApplicationFactory<Startup> fixture)
         public static TheoryData<string> NotAddedEndpoints =>
             new TheoryData<string>()
             {
-                "/AzureAD/Account/AccessDenied",
```

Comment:
So, previously, the Azure AD UI projects used a NullApplicationFactory and instead relied on their own logic to discover the application parts from the related assembly (see https://github.com/dotnet/aspnetcore/pull/31039/files#diff-754ffcfbe4655cc4cca0a446a4f1b0eb3552a8795c136385b6ffad03259e0a7dL199). This API gets invoked from the `AddAzureAD` extension method.

Since the views are compiled into the app assembly and we are no longer using the `NullApplicationFactory`, we don't need to rely on the discovery logic (I removed it in the new PR above).

The controllers don't change though because those are discovered with a feature provider that exists outside of the views story.